Tasks/AzureTestPlanV0/publishAutomatedTests.ts

```diff
@@ -0,0 +1,86 @@
+import tl = require('azure-pipelines-task-lib/task');
+import * as path from 'path';
+import constants = require('./constants');
+
+function publish(testRunner, resultFiles, mergeResults, failTaskOnFailedTests, platform, publishRunAttachments, testRunSystem , failTaskOnFailureToPublishResults, listOfAutomatedTestPoints) {
+    var properties = <{ [key: string]: string }>{};    
+    properties['type'] = testRunner;
+    properties['mergeResults'] = mergeResults;
+    properties['platform'] = platform;
+    properties['publishRunAttachments'] = publishRunAttachments;
+    properties['resultFiles'] = resultFiles;
+    properties['failTaskOnFailedTests'] = failTaskOnFailedTests;
+    properties['failTaskOnFailureToPublishResults'] = failTaskOnFailureToPublishResults;
+    properties['testRunSystem'] = testRunSystem;
+    properties['listOfAutomatedTestPoints'] = listOfAutomatedTestPoints;
+    properties['testPlanId'] = tl.getVariable('TestPlanId');
+
+    tl.command('results.publish', properties, '');
+}
+
+export async function publishAutomatedTestResult(listOfAutomatedTestPoints: string) {
+    try{
+        const testRunner = "JUnit";
```

Comment:
nit: This could be coming from automatedTestType in tbl_reference but for now it should be fine

Reply:
Will address all of the nit changes in next iteration